src/config.js

```diff
@@ -0,0 +1,8 @@
+const postFormatInterceptors = []
+
+module.exports = {
+  _postFormatInterceptors: postFormatInterceptors,
```

Comment:
We generally don't export variables prefixed with underscore. Maybe remove the underscore and then update `src/index.js` to only pass along the `addPostFormatInterceptor` method.